### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by byeze**
Hello, I'm a native spanish speaker from Argentina, I've translated the following sections:

- Milestones
- Translations
- Steering Committee
- Collaborators
- Thank You Notes

### CODE REVIEW

File: README.spanish.md [hunk 1]
```diff
@@ -1095,7 +1095,7 @@ Independent Node.js consultant who works with customers in the USA, Europe, and
 <a href="https://twitter.com/kylemartin_93"><img src="assets/images/twitter-s.png" width="16" height="16"></img></a>
 <a href="https://www.linkedin.com/in/kylemartinnz"><img src="assets/images/linkedin.png" width="16" height="16"></img></a>
 
-Full Stack Developer & Site Reliability Engineer based in New Zealand, interested in web application security, and architecting and building Node.js applications to perform at global scale.
+Desarrollador Full Stack e ingeniero de confiabilidad de sitio con sede en Nueva Zelanda, interesado en la seguridad de aplicaciones web y en la arquitectura y creación de aplicaciones Node.js pque funcionan a escala global
```
Comment (byeze) [inline README.spanish.md:1098]:
```suggestion
Desarrollador Full Stack e ingeniero de confiabilidad de sitio con sede en Nueva Zelanda, interesado en la seguridad de aplicaciones web y en la arquitectura y creación de aplicaciones Node.js que funcionan a escala global
```

**Comment by goldbergyoni** [comment]
@byeze Looks outstanding!! 👯‍♀️😲

@kevynb Going by operations-manual.md I noted that you are handling Spanish, just to let you know that there is some ongoing Spanish translation, led by @desko27,  so better check whether this is in-sync. Let me know if I can help here.

**Comment by desko27** [comment]
@byeze Thank you for contributing!

@goldbergyoni This PR is pointing to `master`, shouldn't it be pointing to [Spanish Translation branch](/goldbergyoni/nodebestpractices/tree/spanish-translation) instead?

**Comment by byeze** [comment]
> @byeze Thank you for contributing!
> 
> @goldbergyoni This PR is pointing to `master`, shouldn't it be pointing to [Spanish Translation branch](/goldbergyoni/nodebestpractices/tree/spanish-translation) instead?

Yes, I pushed in the wrong branch, my bad. 👎

**Comment by goldbergyoni** [comment]
@kevynb How do you propose to handle this?

**Comment by kevynb** [comment]
@byeze I changed the target branch to `spanish-translation`. 

If @desko27 is ok with your changes we will merge it.

File: README.spanish.md [hunk 2]
```diff
@@ -1040,43 +1040,43 @@ Ten en cuenta que con la introducción del nuevo motor V8 junto con los nuevos e
 <br/><br/><br/>
 
 
-# Milestones
+# Metas
 
-To maintain this guide and keep it up to date, we are constantly updating and improving the guidelines and best practices with the help of the community. You can follow our [milestones](https://github.com/i0natan/nodebestpractices/milestones) and join the working groups if you want to contribute to this project
+Para mantener esta guía y tenerla actualizada, estamos constantemente modernizando y mejorando las pautas y las mejores prácticas con la ayuda de la comunidad. Tu puedes seguir nuestras [metas](https://github.com/i0natan/nodebestpractices/milestones) y unirte a los grupos que trabajan si es que quieres contribuir a este proyecto.
```
Comment (desko27) [inline README.spanish.md:1045]:
```suggestion
Para mantener esta guía y tenerla actualizada, estamos constantemente modernizando y mejorando las pautas y las mejores prácticas con la ayuda de la comunidad. Puedes seguir nuestras [metas](https://github.com/i0natan/nodebestpractices/milestones) y unirte a los grupos de trabajo si es que quieres contribuir a este proyecto.
```

File: README.spanish.md [hunk 3]
```diff
@@ -1040,43 +1040,43 @@ Ten en cuenta que con la introducción del nuevo motor V8 junto con los nuevos e
...
 
 <br/>
 
-## Translations
+## Traducciones
 
-All translations are contributed by the community. We will be happy to get any help with either completed, ongoing or new translations!
+Todas las traducciones están contribuidas por la comunidad. Estaremos encantados de obtener ayuda con traducciones completas, en curso o nuevas.
```
Comment (desko27) [inline README.spanish.md:1051]:
```suggestion
Todas las traducciones son contribuidas por la comunidad. Estaremos encantados de obtener ayuda con traducciones completas, en curso o nuevas.
```

File: README.spanish.md [hunk 4]
```diff
@@ -1040,43 +1040,43 @@ Ten en cuenta que con la introducción del nuevo motor V8 junto con los nuevos e
...
 
-### Completed translations
+### Traducciones Completadas
```
Comment (desko27) [inline README.spanish.md:1053]:
```suggestion
### Traducciones completadas
```

File: README.spanish.md [hunk 5]
```diff
@@ -1040,43 +1040,43 @@ Ten en cuenta que con la introducción del nuevo motor V8 junto con los nuevos e
...
 
 <img align="left" width="100" height="100" src="assets/images/members/yoni.png">
 
 [Yoni Goldberg](https://github.com/i0natan)
 <a href="https://twitter.com/goldbergyoni"><img src="assets/images/twitter-s.png" width="16" height="16"></img></a>
 <a href="https://goldbergyoni.com"><img src="assets/images/www.png" width="16" height="16"></img></a>
 
-Independent Node.js consultant who works with customers in the USA, Europe, and Israel on building large-scale Node.js applications. Many of the best practices above were first published at [goldbergyoni.com](https://goldbergyoni.com). Reach Yoni at [@goldbergyoni](https://github.com/goldbergyoni) or [me@goldbergyoni.com](mailto:me@goldbergyoni.com)
+Consultor independiente de Node.js que trabaja con clientes en EE. UU., Europa e Israel en la construcción de aplicaciones Node.js a gran escala. Muchas de las mejores prácticas anteriores se publicaron por primera vez en [goldbergyoni.com](https://goldbergyoni.com). Comuníquese con Yoni a través de [@goldbergyoni](https://github.com/goldbergyoni) o [me@goldbergyoni.com](mailto:me@goldbergyoni.com)
```
Comment (desko27) [inline README.spanish.md:1079]:
```suggestion
Consultor independiente de Node.js que trabaja con clientes en EE. UU., Europa e Israel construyendo aplicaciones Node.js de gran escala. Muchas de las mejores prácticas anteriores se publicaron por primera vez en [goldbergyoni.com](https://goldbergyoni.com). Comuníquese con Yoni a través de [@goldbergyoni](https://github.com/goldbergyoni) o [me@goldbergyoni.com](mailto:me@goldbergyoni.com)
```

File: README.spanish.md [hunk 6]
```diff
@@ -1106,30 +1106,31 @@ Full Stack Developer & Site Reliability Engineer based in New Zealand, intereste
...
 <br/>
 
-## Collaborators
+## Colaboradores
 
-Thank you to all our collaborators! 🙏
+Gracias a todos nuestros colaboradores! 🙏
 
-Our collaborators are members who are contributing to the repository on a regular basis, through suggesting new best practices, triaging issues, reviewing pull requests and more. If you are interested in helping us guide thousands of people to craft better Node.js applications, please read our [contributor guidelines](/.operations/CONTRIBUTING.md) 🎉
+Nuestros colaboradores son miembros que contribuyen regularmente al repositorio, sugiriendo nuevas mejores prácticas, problemas de evaluación, revisión de solicitudes de extracción y más. Si está interesado en ayudarnos a guiar a miles de personas para crear mejores aplicaciones de Node.js, lea nuestras [pautas para contribuyentes](/.operations/CONTRIBUTING.md) 🎉
```
Comment (desko27) [inline README.spanish.md:1117]:
```suggestion
Nuestros colaboradores son miembros que contribuyen regularmente al repositorio, sugiriendo nuevas mejores prácticas, evaluando incidencias, revisando _pull requests_ y más. Si está interesado en ayudarnos a guiar a miles de personas para crear mejores aplicaciones de Node.js, lea nuestras [pautas para contribuyentes](/.operations/CONTRIBUTING.md) 🎉
```

I wouldn't translate "pull requests".

File: README.spanish.md [hunk 7]
```diff
@@ -1106,30 +1106,31 @@ Full Stack Developer & Site Reliability Engineer based in New Zealand, intereste
...
 | <a href="https://github.com/refack" target="_blank"><img src="assets/images/members/refael.png" width="50" height="50"></a> |
 | :--: |
 | [Refael Ackermann](https://github.com/refack) |
 
 <br/>
 
-## Thank You Notes
+## Agradecimientos
+
+Agradecemos cualquier contribución, desde una simple palabra hasta una nueva práctica en nuestra guía. Mira nuestros colaboradores y [documentación para contribuir aquí!](CONTRIBUTORS.md)
```
Comment (desko27) [inline README.spanish.md:1133]:
```suggestion
Agradecemos cualquier contribución, desde una simple palabra hasta una nueva práctica en nuestra guía. ¡Mira nuestros colaboradores y [documentación para contribuir aquí!](CONTRIBUTORS.md)
```

**Comment by desko27** [comment]
Hey @kevynb thank you! The Spanish Readme would be actually fully translated after this is merged. However, the inner contents from sections 6 and 7 are still not done. I was planning to upload section 6 translations some time ago, but in the end I couldn't get the time. I'm not sure if i can get the time for it any soon indeed, but I can definitely keep reviewing translations every now and then.

**Comment by kevynb** [comment]
Alright! No problem, there's no rush! 
@byeze if you still have some time available :) No pressure though

**Comment by byeze** [comment]
Sure! I can also translate those sections.

**Comment by desko27** [comment]
@byeze genial, no hay prisa, por mi parte solo aclarar que si revisas las sugerencias de arriba podemos continuar y hacer merge. En teoría puedes aplicarlas desde esta página, incluida la que pusiste tú mismo al principio 👌

Para las otras secciones siempre puedes abrir otra PR después, de esta manera sumamos el trabajo que ya está hecho. Sobre las sugerencias, cualquier asunto que consideres discutible no dudes en comentármelo, ¡gracias!

**Comment by kevynb** [comment]
@desko27 how do you feel about merging this PR? I didn't understand if that was ok with you. With the upcoming #723, it might be easier to have this one merged.

**Comment by desko27** [comment]
@kevynb I would apply or discuss some of the suggestions at least, but I see no problem about merging it anyway and let the community handle the progressive improvement in the near future, just like #723 👌

**Comment by kevynb** [comment]
@all-contributors add @byeze for translation

**Pull request merged**